### PR TITLE
Move all indigenous creatures to one folder and clean up

### PR DESCRIPTION
**Pull request by TomGoodIdea**
**Data Refactor**: This PR addresses an issue discussed on Discord: https://discord.com/channels/251118043411775489/1057102509656768563/1076951940333846549
## Details
Created a ~~`zoo/`~~ `space wildlife/` folder with the indigenous lifeforms of the Ember Waste, the Acheron (pinging @Saugia), and the Deep Space (pinging @MasterOfGrey).

Also moved the Elenchus data to the `hai/` folder and deleted the unused "Forest (Predator)" government.

### CODE REVIEW

File: data/zoo/indigenous.txt
Comment (Zitchas) [inline data/zoo/indigenous.txt:1]: The indigenous creatures of the Ember Waste have reasons for being where they are. Please do not move this file.

**Review by Zitchas — CHANGES_REQUESTED** [review]
Overall this is probably a good idea. However, I would recommend using a better name than "zoo." Perhaps "space wildlife" or just merely "wildlife."

That being said, there are reasons for the indigenous wildlife of the Ember Waste to be stored where they are, and it would be best to leave them there.

File: data/zoo/indigenous.txt
Comment (TomGoodIdea) [reply data/zoo/indigenous.txt:1]: Could you please explain the reasons?

File: data/governments.txt [hunk 1]
```diff
@@ -268,36 +268,6 @@ government "Forest (Prey)"
 		destroy 0
 		atrocity 0
 
-government "Forest (Predator)"
```
Comment (warp-core) [inline data/governments.txt:271]: This government is still referred to in the `"Pirate (Devil-Run Gang)"` definition, you'll need to remove that line as well.
https://github.com/endless-sky/endless-sky/blob/a45fb00b7f03f22844b81452546eaecd03fddae5/data/governments.txt#L953

**Comment by Amazinite** [comment]
I don't think that it makes sense to throw all the space life stuff into the same folder. As Zitchas has noted, the void sprites aren't just window dressing, they have story importance. The other space life groups only lack story importance insofar as they're much newer groups.

I'd say strip this down to just the removal of the unused government, leaving everything else where it is.

**Comment by TomGoodIdea** [comment]
Alright, I'll close this PR and open a new one for the unused government removal.

**Pull request closed without merging**